lib/compute/jenkins-main-node.ts

```diff
@@ -350,7 +350,7 @@ export class JenkinsMainNode {
       // Commands are fired one after the other but it does not wait for the command to complete.
       // Therefore, sleep 60 seconds to wait for jenkins to start
       // This allows jenkins to generate the secrets files used for auth in jenkins-cli APIs
-      InitCommand.shellCommand('sleep 65'),
+      InitCommand.shellCommand('sleep 60'),
```

Comment:
Are we changing the sleep time to support the removal of multiple-scms plugin?

Reply:
Not this is just to trigger the redeployment in the internal stack. Just change in plugin won't trigger a new EC2 with new config. This was 60 before, so just resetting to what it was.